docs/core/whats-new/dotnet-9/overview.md

```diff
@@ -19,169 +17,47 @@ This article has been updated for .NET 9 Preview 3.
+## .NET Aspire
 
-Determining the type of an object requires a call into the runtime, which comes with a performance penalty. When the type of an object needs to be checked, the 64-bit JIT compiler emits this call for the sake of correctness (compilers usually cannot rule out any possibilities, even if they seem improbable). However, if PGO data suggests an object is likely to be a specific type, the 64-bit JIT compiler now emits a *fast path* that cheaply checks for that type, and falls back on the slow path of calling into the runtime only if necessary.
+.NET Aspire is an opinionated, cloud-ready stack for building observable, production ready, distributed applications.​ .NET Aspire is delivered through a collection of NuGet packages that handle specific cloud-native concerns, and is available in preview for .NET 9. For more information, see [.NET Aspire (Preview)](/dotnet/aspire).
 
-#### Arm64 vectorization in .NET libraries
+## ASP.NET Core
 
-A new `EncodeToUtf8` implementation takes advantage of the 64-bit JIT compiler's ability to emit multi-register load/store instructions on Arm64. This behavior allows programs to process larger chunks of data with fewer instructions. .NET apps across various domains should see throughput improvements on Arm64 hardware that supports these features. Some [benchmarks](https://github.com/dotnet/perf-autofiling-issues/issues/27114) cut their execution time by more than half.
+ASP.NET Core includes improvements to ... For more information, see [What's new in ASP.NET Core 9.0](/aspnet/core/release-notes/aspnetcore-9.0).
 
-## .NET SDK
+## .NET MAUI
 
-- [Unit testing](#unit-testing)
-- [.NET tool roll-forward](#net-tool-roll-forward)
+.NET MAUI includes ... For more information, see [What's new in .NET MAUI for .NET 9](/dotnet/maui/whats-new/dotnet-9).
 
-### Unit testing
+## EF Core
 
-This section describes the updates to unit testing in .NET 9: running tests in parallel, and terminal logger test output.
+Entity Framework Core includes improvements to ... For more information, see [What's New in EF Core 9](/ef/core/what-is-new/ef-core-9.0/whatsnew).
 
-#### Run tests in parallel
+<!--
 
-In .NET 9, `dotnet test` is more fully integrated with MSBuild. Because MSBuild supports [building in parallel](/visualstudio/msbuild/building-multiple-projects-in-parallel-with-msbuild), you can run tests for the same project across different target frameworks in parallel. By default, MSBuild limits the number of parallel processes to the number of processors on the computer. You can also set your own limit using the [-maxcpucount](/visualstudio/msbuild/building-multiple-projects-in-parallel-with-msbuild#-maxcpucount-switch) switch. If you want to opt out of the parallelism, set the `TestTfmsInParallel` MSBuild property to `false`.
+## C# 12
 
-#### Terminal logger test display
+C# 12 shipped with the .NET 8 SDK. For more information, see [What's new in C# 12](../../../csharp/whats-new/csharp-12.md).
```

Comment:
I do have the start on What's new for C# 13 in place (more will be added in the next week or two):

https://github.com/dotnet/docs/blob/main/docs/csharp/whats-new/csharp-13.md

It will ship with .NET 9 SDK.